docs/user/experiments.rst

```diff
@@ -119,24 +116,17 @@ Additional arguments for `run_experiment`:
 Running Experiments with TensorFlow and GPU
 ===========================================
 
-To run experiments in the TensorFlow tree of garage with the GPU enabled, set the flags use_tf and use_gpu to True when calling `run_experiment`, as shown in the code below:
+Using `tensorflow-gpu`, TensorFlow has GPU enabled by default. Set the flags `force_cpu` to True to disable GPU, as shown in the code below:
```

Comment:
Maybe a nitpick, but I suggest this to be consistent with installation docs
```suggestion
If you installed the `garage[gpu]` extra package, TensorFlow will use GPU by default. To disable GPU, set the flag `force_cpu` to True, as shown in the code below:
```

Reply:
Thanks for suggestions! Updated.